src/cosalib/aws.py

```diff
@@ -116,9 +115,9 @@ def aws_run_ore(build, args):
 
     # This matches the Container Linux schema:
     # https://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json
-    ami_data = build.meta.get("aws", [])
+    ami_data = build.meta.get("amis", [])
     ami_data.append({
-        'name': args.region,
+        'name': args.region[0],
```

Comment:
I think this is fine, but to make sure the `nargs='+'` will ensure that it's always a list of strings even when only one value for the parameter is given correct?